RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -1,3 +1,74 @@
+void bedtime_still()
+{
+	//quickly use up all remaining uses of Nash Crosby's Still during bedtime
+	if(!stillReachable())
+	{
+		return;		//we can not reach the still
+	}
+	while(stills_available() > 0)	//spend remaining still uses
+	{
+		item target = $item[none];
+		
+		//first try to get at least 1 each of each of the imrpoved booze if possible
+		foreach it in $items[bottle of Calcutta Emerald, bottle of Lieutenant Freeman, bottle of Jorge Sinsonte, bottle of Definit, bottle of Domesticated Turkey, boxed champagne, bottle of Pete\'s Sake]
```

Comment:
I would remove Pete's Sake from here and the lower all items foreach too. It's not something we'd want to use in-run since it's for Sea adventuring purposes so lets err on the side of caution.

Reply:
fixed